src/src/PluginStructs/P110_data_struct.h

```diff
@@ -7,11 +7,11 @@
 # define P110_DEBUG       // Enable debugging output (INFO loglevel)
 # define P110_DEBUG_DEBUG // Enable extended debugging output (DEBUG loglevel)
 
-# ifdef LIMIT_BUILD_SIZE
+# if defined(LIMIT_BUILD_SIZE) || !defined(BUILD_NO_DEBUG)
```

Comment:
Is the `||` correct here?

Reply:
Maybe the `!` shouldn't be there

Reply:
Nah that `!` is OK, but I think it might need to be `&&` instead of `||`

Another thing to look into is the horrible flag it is wrapping: `P110_DEBUG_DEBUG` is probably not meant to debug the debug code?

Reply:
Renamed the flags